From 8b8a72d83065cade979a84bf0903fad472b18585 Mon Sep 17 00:00:00 2001
From: syphyr <syphyr@gmail.com>
Date: Wed, 6 Feb 2019 21:07:39 +0100
Subject: [PATCH] Bump Security Patch Level to 2019-02-05

Implemented:
============

CVE-2019-1987	A-118143775 [2] RCE 	Critical 	7.0, 7.1.1, 7.1.2, 8.0, 8.1, 9
CVE-2019-1991	A-110166268 	RCE 	Critical 	7.0, 7.1.1, 7.1.2, 8.0, 8.1, 9
CVE-2019-1992	A-116222069 	RCE 	Critical 	7.0, 7.1.1, 7.1.2, 8.0, 8.1, 9
CVE-2019-1995	A-32589229 [2] 	ID 	High 		7.0, 7.1.1, 7.1.2, 8.0, 8.1, 9
CVE-2019-1997	A-117508900 	ID 	High 		7.0, 7.1.1, 7.1.2, 8.0, 8.1, 9

Not Implemented:
================

CVE-2017-17760 	A-78029030* 	RCE 	High 		7.0, 7.1.1, 7.1.2, 8.0, 8.1, 9
CVE-2018-5268 	A-78029634* 	RCE 	High 		7.0, 7.1.1, 7.1.2, 8.0, 8.1, 9
CVE-2018-5269 	A-78029727* 	RCE 	High 		7.0, 7.1.1, 7.1.2, 8.0, 8.1, 9
CVE-2017-18009 	A-78026242* 	ID 	Moderate 	7.0, 7.1.1, 7.1.2, 8.0, 8.1, 9

No impact:
==========

CVE-2019-1986 	A-117838472 [2] RCE 	Critical 	9
CVE-2019-1988 	A-118372692 	RCE 	Critical 	8.0, 8.1, 9
CVE-2019-1993 	A-119819889 	EoP 	High 		8.0, 8.1, 9
CVE-2019-1994 	A-117770924 	EoP 	High 		8.0, 8.1, 9
CVE-2019-1996 	A-111451066 	ID 	High 		8.0, 8.1, 9
CVE-2019-1998 	A-116055338 [2] DoS 	High 		9

Change-Id: Ib7ade3ab4a1780d829aba3bad9db04ffee6799d1
---
 core/version_defaults.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/version_defaults.mk b/core/version_defaults.mk
index 546e6fb..34e983c 100644
--- a/core/version_defaults.mk
+++ b/core/version_defaults.mk
@@ -131,7 +131,7 @@ ifeq "" "$(PLATFORM_SECURITY_PATCH)"
     #  It must be of the form "YYYY-MM-DD" on production devices.
     #  It must match one of the Android Security Patch Level strings of the Public Security Bulletins.
     #  If there is no $PLATFORM_SECURITY_PATCH set, keep it empty.
-    PLATFORM_SECURITY_PATCH := 2019-01-05
+    PLATFORM_SECURITY_PATCH := 2019-02-05
 endif
 
 ifeq "" "$(PLATFORM_BASE_OS)"
-- 
2.7.4

